src/core/cgroup.c

```diff
@@ -189,6 +190,268 @@ void cgroup_context_init(CGroupContext *c) {
         };
 }
 
+int cgroup_context_copy(CGroupContext *dst, const CGroupContext *src) {
+        struct in_addr_prefix *i;
+        char *iface;
+        int r;
+
+        assert(src);
+        assert(dst);
+
+        dst->cpu_accounting = src->cpu_accounting;
+        dst->io_accounting = src->io_accounting;
+        dst->blockio_accounting = src->blockio_accounting;
+        dst->memory_accounting = src->memory_accounting;
+        dst->tasks_accounting = src->tasks_accounting;
+        dst->ip_accounting = src->ip_accounting;
+
+        dst->memory_oom_group = dst->memory_oom_group;
+
+        dst->delegate = src->delegate;
+        dst->delegate_controllers = src->delegate_controllers;
+        dst->disable_controllers = src->disable_controllers;
+        dst->delegate_subgroup = src->delegate_subgroup;
+
+        dst->cpu_weight = src->cpu_weight;
+        dst->startup_cpu_weight = src->startup_cpu_weight;
+        dst->cpu_quota_per_sec_usec = src->cpu_quota_per_sec_usec;
+        dst->cpu_quota_period_usec = src->cpu_quota_period_usec;
```

Comment:
this is particularly weird, since you start overcommit time here, if you just copy this.